Document Satellite Ansible Content Collections (#228)
theforeman/foreman-documentation: guides/common/assembly_using-satellite-ansible-content-collections.adoc

ifdef::context[:parent-context: {context}]

[id="using-satellite-ansible-content-collections_{context}"]
== Using {Project} Ansible Content Collections

> The {Project} Ansible Content Collections is a set of Ansible modules and plug-ins to interact with {Project}.

Reviewer: ```suggestion
The {Project} Ansible Content Collections is a set of Ansible modules and plug-ins that you can use to manage {Project}.
```

Author: Done

ifdef::context[:parent-context: {context}]

[id="using-satellite-ansible-content-collections_{context}"]
== Using {Project} Ansible Content Collections

> The {Project} Ansible Content Collections is a set of Ansible modules and plugins to interact with {Project}. The modules are based on the upstream https://github.com/theforeman/foreman-ansible-modules/[Foreman Ansible Modules Collection].

Author: Fixed.

> Using {Project} Ansible Content Collections to interact with {Project} requires:

Author: Removed.
